Packs/Core/Playbooks/playbook-Impossible_Traveler.yml

```diff
@@ -1143,19 +1143,6 @@ inputs:
   required: false
   description: A comma separated list of CIDR that are allowed to be used across long distances.
   playbookInputQuery:
-- key: username
-  value:
-    complex:
-      root: alert
-      accessor: username
```

Comment:
why you removed this one?

Reply:
because of validation error saying this input is declared but not used, should we keep it anyway?